vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -37337,11 +37336,12 @@ unit:PERCENT-PER-DecaKiloCount
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "percent per one hundred thousand" ;
 .
-unit:PERCENT-PER-DektoKiloCount
```

Comment:
Is it possible `unit:PERCENT-PER-DektoKiloCount` has just been renamed without `qudt:deprecated`/`qudt:isReplacedBy` in this PR?